sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/ExpressionCompiler.java

```diff
@@ -564,6 +564,17 @@ A condition case (CASE WHEN p1 THEN v1 ... ELSE e END) has an odd number of
                         "_" + keyword + type1.nullableSuffix();
                 return new DBSPApplyExpression(node, functionName, type, ops.get(1));
             }
+            case LIKE:
+            case SIMILAR: {
+                String functionName = call.getKind().toString().toLowerCase() + ops.size();
+                if (call.operands.size() == 2) {
```

Comment:
Maybe add comments explaining what the 2 and 3 operand versions represent? I assume its the case with and without ESCAPE?

Reply:
I will add documentation for the escape argument in the next PR, which will also fix main after Chase's latest pushes.